content/integrations/nxlog.md

```diff
@@ -1,9 +1,28 @@
 ---
-title: NXLog Log collection
-kind: Documentation
+title: NXLog
+kind: integration
 description: "Configure NXLog to gather logs from your host, containers & services."
+short_description: "Configure NXLog to gather logs from your host, containers & services."
```

Comment:
serial comma here and in the line above. Configure NXLog to gather logs from your host, containers, & services.